### DISCUSSION TITLE
Disable number formatting

### DISCUSSION THREAD

**Discussion by mattjamison** [graphia-app/graphia#51] [Q&A]
Is there any way to disable the formatting of numeric attribute fields?  What I'm seeing is that any numeric fields are parsed into scientific notation, commas added, etc., even if I've explicitly quoted the input to try to coerce it into a string?  Postcodes of "90210" for example, become "90,210" in the attribute table.   I'm using the latest version, opening a dot language file.

 I'm very new to dot language and your awesome tool, so please forgive me if I'm missing something simple.  Thanks!

**Comment by timangus** [comment]
The type of an attribute is determined entirely by the values of said attribute. I have occasionally been asked to provide a means to allow the user to explicitly set the types of attributes, but this isn't necessarily a 5 minute job so isn't something I have got around to yet. As a work around you could add some characters to your values to disambiguate, e.g. instead of "90210" you would make it "90210 postcode". You could do a similar thing in Graphia itself using an Attribute Synthesis transform:

![Screenshot from 2022-08-29 14-05-10](https://user-images.githubusercontent.com/3159586/187208151-e3891e0f-bc0c-45df-b57b-be37e85b2af8.png)
